source/com.microsoft.tfs.client.common.ui/src/com/microsoft/tfs/client/common/ui/controls/connect/ServerTypeSelectControl.java

```diff
@@ -180,16 +302,31 @@ public ICommandExecutor getNoErrorDialogCommandExecutor() {
     }
 
     public URI getServer() {
+        if (this.serverURI == null) {
+            return URIUtils.VSTS_ROOT_URL;
+        }
         return this.serverURI;
     }
 
     public void setServer(final URI serverURI) {
+        setServer(serverURI, true);
+    }
+
+    private void setServer(URI serverURI, final boolean setCombo) {
         ignoreServerChangeEvents = true;
-        serverControl.setServerURI(serverURI);
+        if (serverURI != null && URIUtils.VSTS_ROOT_URL.equals(serverURI)) {

```

Comment:
In the setter, I convert it to null because I want the combo to be cleared. Within this control, null is the same as the VSTS radio button being selected.